src/pages/event/event.tsx

```diff
@@ -0,0 +1,107 @@
+import styled from "@emotion/styled";
+import React, { FunctionComponent, useState } from "react";
+import { compareDesc, isFuture, isPast } from "date-fns";
+import { Helmet } from "react-helmet";
+import { ResourceEvent, EventProps } from "../../components/UI/ResourceEvent";
+import { importAll } from "../../common/utils/importAll";
+import { Pagination } from "@govtechsg/tradetrust-ui-components";
+
+let events = importAll(require.context("../../../cms/event/", false, /\.md$/)) as EventProps[];
+
+const CategoryFilter = styled.div`
+  h5 {
+    color: #b4bcc2;
+
+    &:hover {
+      color: #454b50;
+    }
+
+    &.active {
+      color: #454b50;
+    }
+  }
+`;
+
+const getSortedByDateDesc = (items: any[]) => {
+  items.sort((a, b): number => {
+    return compareDesc(new Date(a.attributes.date), new Date(b.attributes.date));
+  });
+
+  return items;
+};
```

Comment:
shift this to a common utility area, then import and use from there?

Reply:
done. shifted it to src/utils/index.ts.